wrappers/services.go

```diff
@@ -198,6 +224,17 @@ func AddSnapServices(s *snap.Info, inter interacter) (err error) {
 			written = append(written, path)
 		}
 
+		if app.Timer != nil {
+			content := generateSnapTimerFile(app)
+			path := app.Timer.File()
+			os.MkdirAll(filepath.Dir(path), 0755)
+			if err := osutil.AtomicWriteFile(path, content, 0644, 0); err != nil {
+				return err
+			}
+			written = append(written, path)
+			continue
```

Comment:
Otherwise there's a `sysd.Enable()` call just below. Once enabled the service would automatically start after reboot, but we only want it to be activated by the timer.

Reply:
Can you add a comment in the code please?